### PR TITLE
Make Parameter Names PascalCase

### PR DESCRIPTION
**Pull request by aarani**
@knocte

### CODE REVIEW

**Comment by knocte** [comment]
Parameter names? I never mentioned parameters. Parameters should be camelCase. I mentioned methods, properties and public fields.

**Comment by aarani** [comment]
I meant Request Parameters

File: TeleSharp.Generator/Program.cs [hunk 1]
```diff
@@ -146,7 +146,7 @@ static void Main(string[] args)
                     string fields = "";
                     foreach (var tmp in c.Params)
                     {
-                        fields += $"        public {CheckForFlagBase(tmp.type, GetTypeName(tmp.type))} {CheckForKeyword(tmp.name)} " + "{get;set;}" + Environment.NewLine;
+                        fields += $"        public {CheckForFlagBase(tmp.type, GetTypeName(tmp.type))} {CheckForKeywordAndPascalCase(tmp.name)} " + "{get;set;}" + Environment.NewLine;
```
Comment (aarani) [inline TeleSharp.Generator/Program.cs]: Public Field

File: TeleSharp.Generator/Program.cs [hunk 2]
```diff
@@ -80,7 +80,7 @@ static void Main(string[] args)
                     string fields = "";
                     foreach (var tmp in c.Params)
                     {
-                        fields += $"     public {CheckForFlagBase(tmp.type, GetTypeName(tmp.type))} {CheckForKeyword(tmp.name)} " + "{get;set;}" + Environment.NewLine;
+                        fields += $"     public {CheckForFlagBase(tmp.type, GetTypeName(tmp.type))} {CheckForKeywordAndPascalCase(tmp.name)} " + "{get;set;}" + Environment.NewLine;
```
Comment (aarani) [inline TeleSharp.Generator/Program.cs]: Public Field

**Comment by knocte** [comment]
Ah ok. Can you add a last commit that changes the generated files so that we can see the effects of the changes on the generator in this PullRequest?

**Comment by aarani** [comment]
@knocte With Same Old Layer 66 ? or New One ?

**Comment by knocte** [comment]
Same layer. And after we merge this you can add a new PullRequest to upgrade the layer number.

**Comment by aarani** [comment]
Ok it will took some minute for my server to batch format all files

**Comment by aarani** [comment]
I had to squash this cause i had to update all hand-writed code too so spliting it would make bunch of commits unbuildable

**Comment by aarani** [comment]
@knocte

**Comment by knocte** [comment]
Good stuff! I only see one conversion missing: lists -> Lists, can you fix this one?

**Comment by aarani** [comment]
@knocte  It's not generated code, I have another way to get rid of it

**Comment by aarani** [comment]
but let me merge this first

**Comment by knocte** [comment]
Ok sure, merging.

**Pull request merged**